src/sentry/search/events/constants.py

```diff
@@ -13,6 +13,7 @@
 ISSUE_ALIAS = "issue"
 ISSUE_ID_ALIAS = "issue.id"
 RELEASE_ALIAS = "release"
+SEMVER_ALIAS = "sentry.semver"
```

Comment:
nit: should we use invalid tag characters for the alias so there's 0 chance a user can submit it as a tag accidentally?
eg. `sentry[semver]`

Reply:
I'm not too worried about this, it's a temporary name and I kind of doubt it is currently in use